OptionSelector/OptionSelectorDetailTableViewController.m

```diff
@@ -0,0 +1,115 @@
+//
+//  OptionSelectorDetailTableViewController.m
+//  OptionSelector
+//
+//  Created by Mesfin Bekele Mekonnen on 8/10/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "OptionSelectorDetailTableViewController.h"
+
+
+@interface OptionSelectorDetailTableViewController ()
+
+
+@end
+
+@implementation OptionSelectorDetailTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+}
+
+#pragma mark - Table view data source
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+    return 1;
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+    return self.type.options.count;
+}
+
+-(void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath{
+    
+    self.type.selection = self.type.options[indexPath.row];
+    [self.delegate optionSelectorDetailTableViewController:self didSelectType:self.type];
+    
+    NSArray *indexes = [tableView visibleCells];
```

Comment:
This variable name is misleading, as it is an array of `UITableViewCell`s, not an array of indexes (`int`s). So a better name would be `cells` or `allCells`.